opencensus/log/__init__.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import namedtuple
+from copy import copy
+import logging
+
+from opencensus.trace import execution_context
+
+
+_meta_logger = logging.getLogger(__name__)
```

Comment:
@c24t do you think we would encourage the library and extensions to leverage_meta_logger for internal logs (from the lib/exts)?

Reply:
This is the same convention we use for (internal) logging elsewhere, I just didn't name this one `logger` since I thought it might confuse users. I think `logger` is usually fine.